tools/__tasks__/validate-head/javascript-fix.js

```diff
@@ -0,0 +1,23 @@
+const execa = require('execa');
+const getChangedFiles = require('../lib/get-changed-files');
+
+module.exports = {
+    description: 'Fix committed linting errors',
```

Comment:
It's not attached to a hook, so I don't think it's worth prompting. A dev would only run `make fix-commits`  if they wanted files fixed, it's literally the only thing this task does. 

It should be okay for this task to fix lint issues in uncommitted files. I can't think of a reason why someone would want to prevent this? It won't change the logic, and if they want to commit their changes in the future, they will need to fix lint issues anyway.